code/modules/antagonists/wizard/equipment/artefact.dm

```diff
@@ -377,7 +377,10 @@
 /obj/item/warpwhistle/proc/end_effect(mob/living/carbon/user)
```

Comment:
heard complaints that warp whistle was broken--might be related

Reply:
when?